### PR TITLE
 Monitor progress of direct CDC 

### LINKED ISSUE #1513: Document how users can tell whether RW catches up with the upstream PG/MySQL when using direct CDC
We are still testing the direct CDC for these two databases.

Once we find a good way to monitor the progress, we'll update it here.

cc: @cyliu0 @StrikeW

### PR DESCRIPTION
**Pull request by WanYixian**
## Info

- **Description**

  -  Add how to monitor progress of direct CDC for PG and MySQL

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1513

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/guides/ingest-from-mysql-cdc.md [hunk 1]
```diff
@@ -432,3 +432,60 @@ And this it the output of `DESCRIBE supplier;`
...
+    ----------+----+-------------------+-----------+--------------------------------------------------
+    3        |  5 | t                 |         4 | {"Postgres": {"lsn": 4558482960, "txid": 35853}}
+    (1 row)
+    ```
+
+### For real-time data
+
+RisingWave stores source offset in the internal state table of source executor. You can check the current consumed offset by checking this table and comparing it with the upstream database's log offset.
+
+You can execute the following SQL on upstream MySQL to get the current binlog offset.
```
Comment (StrikeW) [inline docs/guides/ingest-from-mysql-cdc.md]: `SHOW MASTER STATUS` only available for mysql version less than 8.4, suggest to add a note.

File: docs/guides/ingest-from-postgres-cdc.md [hunk 2]
```diff
@@ -432,3 +432,64 @@ And this it the output of `DESCRIBE supplier;`
...
+    (1 row)
+    ```
+
+### For real-time data
+
+RisingWave stores source offset in the internal state table of source executor. You can check the current consumed offset by checking this table and comparing it with the upstream database's log offset.
+
+The Postgres connector commits offsets to the upstream database, allowing Postgres to free up space used by Write-Ahead Log (WAL) files. This offset commitment happens during checkpoint commits in the CDC source. If there is high checkpoint point latency, WAL files may accumulate on the upstream server.
+
+To check for WAL accumulation on the upstream Postgres server, run this SQL query:
```
Comment (StrikeW) [inline docs/guides/ingest-from-postgres-cdc.md]: should run on upstream PG.

**Review by StrikeW — APPROVED** [review]
Rest lgtm

File: docs/guides/ingest-from-mysql-cdc.md [hunk 3]
```diff
@@ -432,3 +432,60 @@ And this it the output of `DESCRIBE supplier;`
...
+To observe the progress of direct CDC for MySQL, use the following methods:
+
+### For historical data
+
+Historical data needs to be backfilled into the table. You can check the internal state of the backfill executor as follows:
+
+1. Create a table to backfill historical data:
+
+    ```sql
+    CREATE TABLE t3 (id INTEGER, v1 TIMESTAMP WITH TIME ZONE, PRIMARY KEY(id)) FROM pg_source TABLE 'public.t3';
```
Comment (lmatz) [inline docs/guides/ingest-from-mysql-cdc.md]: the page is under mysql, but the example is PG

File: docs/guides/ingest-from-mysql-cdc.md [hunk 4]
```diff
@@ -432,3 +432,60 @@ And this it the output of `DESCRIBE supplier;`
...
+    SHOW INTERNAL TABLES;
+    ```
+
+    Output:
+
+    ```
+    Name
+    ---------------------------------
+    __internal_t3_3_streamcdcscan_4
+    __internal_pg_source_1_source_2
```
Comment (lmatz) [inline docs/guides/ingest-from-mysql-cdc.md]: ditto

**Review by lmatz — APPROVED** [review]
rest LGTM

**Pull request merged**